src/platform/intel/cavs/include/cavs/lib/memory.h

```diff
@@ -19,14 +19,22 @@
 
 #define EBB_SEGMENT_SIZE		EBB_BANKS_IN_SEGMENT
 
+#if CONFIG_LP_MEMORY_BANKS
 #define PLATFORM_LPSRAM_EBB_COUNT	CONFIG_LP_MEMORY_BANKS
+#else
+#define PLATFORM_LPSRAM_EBB_COUNT	0
+#endif
```

Comment:
@mrajwa , the preprocessor replaces undefined macros with 0 so I don't see when or how the newer and more convoluted code behaves differently from the previous code, can you explain? Besides moving any `-Wundef` warning to the previous line I couldn't find any difference.